From afa5ac1967dafbc7f147d53301782ceedf798140 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Thu, 01 Apr 2021 14:16:02 +0300
Subject: [PATCH] [SW-28013] ppboot: increasing ppboot sram size

In order to add iatu initialization function, the ppboot sram size
should be increased.

Change-Id: I7081308861c7c38320f4dafeb8a059744c30eb04
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/gaudi/common.h b/gaudi/common.h
index b99b9cc..e0cf8f9 100644
--- a/gaudi/common.h
+++ b/gaudi/common.h
@@ -7,7 +7,7 @@
 #ifndef COMMON_GAUDI_H_
 #define COMMON_GAUDI_H_
 
-#define PPBOOT_SIZE			0x4400		/* 17KB */
+#define PPBOOT_SIZE			0x5000		/* 20KB */
 #define PREBOOT_SIZE			0x20000		/* 128KB */
 #define SRAM_SIZE			0x1400000	/* 20MB */
 
